.vsts-ci.yml

```diff
@@ -47,164 +47,286 @@ variables:
+            os: windows
+        steps:
+          - task: 1ES.PublishPipelineArtifact@1
+            displayName: Publish Build Config
+            inputs:
+              targetPath: $(Build.SourcesDirectory)\eng\buildConfiguration
+              artifactName: buildConfiguration
+      - template: /eng/build.yml@self
+        parameters:
+          agentOs: Windows_NT
+          pool:
+            ${{ if eq(variables['System.TeamProject'], 'public') }}:
+              name: $(DncEngPublicBuildPool)
+              image: 1es-windows-2022-open
+              os: windows
+            ${{ if ne(variables['System.TeamProject'], 'public') }}:
+              name: $(DncEngInternalBuildPool)
+              image: 1es-windows-2022
+              os: windows
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
-            name: $(DncEngPublicBuildPool)
-            demands: ImageOverride -equals 1es-windows-2022-open
+            helixTargetQueue: Windows.Amd64.VS2022.Pre.Open
           ${{ if ne(variables['System.TeamProject'], 'public') }}:
-            name: $(DncEngInternalBuildPool)
-            demands: ImageOverride -equals 1es-windows-2022
-        strategy:
-          matrix:
-            Build_Debug:
-              _BuildConfig: Debug
-              _PublishArgs: ''
-              _SignType: test
+            helixTargetQueue: Windows.Amd64.VS2022.Pre
+          variables:
+          - name: _BuildConfig
+            value: Release
+          - name: _PublishArgs
+            value: '-publish /p:DotNetPublishUsingPipelines=true'
+          - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+            - name: _SignType
+              value: test
+            - name: _Test
+              value: -test
+          - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+            - name: _SignType
+              value: real
+            - name: _Test
+              value: ''
+      - template: /eng/common/templates-official/job/source-build.yml@self
+        parameters:
+          platform:
+            name: 'Managed'
+            container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
+      - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+        - template: /eng/build.yml@self
+          parameters:
+            agentOs: Windows_NT_FullFramework
+            pool:
+              ${{ if eq(variables['System.TeamProject'], 'public') }}:
+                name: $(DncEngPublicBuildPool)
+                image: 1es-windows-2022-open
+                os: windows
+              ${{ if ne(variables['System.TeamProject'], 'public') }}:
+                name: $(DncEngInternalBuildPool)
+                # Should this be windows.vs2022preview.amd64?
+                image: 1es-windows-2022
+                os: windows
```

Comment:
If you go to https://helix.dot.net/ and find `windows.vs2022preview.amd64`, you can click on it and it'll show you the details of the image. However, if you also find `1es-windows-2022` and click on it, it doesn't have any details. So... ❓❓❓